### ISSUE TITLE
The same result for all the attributes.

### ISSUE THREAD

**Issue by acecreamu** [LynnHo/AttGAN-Tensorflow#7]
As written in the title, I obtain a row of the same images without any changes regardless to the attribute (column). I use custom data-set organized as CelebA. Could you give an advise, what may cause it?

**Comment by LynnHo** [comment]
@acecreamu I need more information, which model and command did you use?

**Comment by acecreamu** [comment]
@LynnHo  I do all the steps from instruction for CelebA 128x128. Structure of a dataset and an attribute txt file is the same as for CelebA. 
I run code using:
```
CUDA_VISIBLE_DEVICES=0 \
python train.py \
--img_size 128 \
--shortcut_layers 1 \
--inject_layers 1 \
--experiment_name 128_shortcut1_inject1_none
```
My dataset is very small, 2222 images of faces, 3 attribute for each.

**Comment by LynnHo** [comment]
@acecreamu This code is not suitable for other datasets. However, you can try to modify "data.py" to fit your dataset. Besides, you should also change the default attributes in "train.py".

**Comment by acecreamu** [comment]
@LynnHo  Thank you, I understand it. I did exactly these steps. Eventually it works, but my question is about the output - why change of attribute doesn't influence an output image?

**Comment by LynnHo** [comment]
@acecreamu It may be caused by many reasons. Can you post your loss curves to check whether the losses are abnormal. You can also post the results during training. Besides, what are the attributes of your dataset.

**Comment by vcodreanu** [comment]
@acecreamu I am having a similar issue where for every attribute modified I am getting the same output image. Could you indicate what you did to go around this issue? I am also using a different dataset, with 14 attributes in total.

**Comment by acecreamu** [comment]
@LynnHo Sorry, I don't know how to visualize losses. But there is what I noticed: 
(I use CelebA dataset with additional custom attribute) It introduce differences during first 2-3 epochs, but then goes to the same results for all the attributes. For example:
Epoch 2:
![image](https://user-images.githubusercontent.com/40887465/44944586-de118a00-add9-11e8-82b8-75b27214e860.png)
Epoch 20:
![image](https://user-images.githubusercontent.com/40887465/44944599-0ef1bf00-adda-11e8-8e86-cef1cdfca248.png)

**Comment by LynnHo** [comment]
@acecreamu I am sorry I can't tell you the true reason for this situation, it may be caused by many reasons. Here is the example for visualizing the losses, you may have a knowledge of tensorboard usage. 

```console
CUDA_VISIBLE_DEVICES='' \
tensorboard \
--logdir ./output/128_shortcut1_inject1_none/summaries \
--port 6006
```
